nav2_regulated_pure_pursuit_controller/src/regulated_pure_pursuit_controller.cpp

```diff
@@ -271,7 +494,49 @@ geometry_msgs::msg::TwistStamped RegulatedPurePursuitController::computeVelocity
     }
 
     // Apply curvature to angular velocity after constraining linear velocity
-    angular_vel = linear_vel * regulation_curvature;
+    if (!params_->use_dynamic_window) {
+      angular_vel = linear_vel * regulation_curvature;
+    } else {
+      // compute optimal path tracking velocity commands
+      // considering velocity and acceleration constraints (DWPP)
+      const double regulated_linear_vel = linear_vel;
+      geometry_msgs::msg::Twist current_speed;
+      if (params_->velocity_feedback == "CLOSED_LOOP") {
```

Comment:
Shouldn't be be made of use more generally if we're enabling open vs closed loop? This is not the only place speed is utilized